### ISSUE TITLE
RSS feed for news?

### ISSUE THREAD

**Issue by Blaisorblade** [ps-mr/website#27]
Doesn't Jekyll provide RSS support, or is that just part of Octopress? I certainly didn't do much to activate it for my blog (http://blaisorblade.github.io/).
(Not that *I* use RSS, but I hear some still do).

**Comment by Toxaris** [comment]
Yes, it would be reasonably easy to have an RSS feed. However, someone (Jona? Paolo?) told me that RSS is dead, so I'm not inclined to put any work into it at all. If I get correct on the "RSS is dead" part, I can reconsider.

**Comment by Blaisorblade** [comment]
I've little clue whether RSS is dead; I agree on the rest.
@b-studios, do you know ?

**Comment by Toxaris** [comment]
One benefit of RSS is that we can feed it into a rss2twitter gateway.

**Comment by Toxaris** [comment]
Adapting the template from https://github.com/snaptortoise/jekyll-rss-feeds should be easy enough.
